src/components/Timer/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './styles.css'
+import PropTypes from 'prop-types'
+
+const Timer = ({ expirationDate }) => {
+  return (
+    <div className='timer'>
+      <img src='https://placehold.co/15x15' alt='clock' className='clock' />
```

Comment:
Decoratives images such as this one should have an empty `alt` attribute `alt=''`.(an empty alt also has an implicit `role='presentation'`). This is mainly for assistive technologies (screen readers and such), and to not go overboard with alternatives texts, they appear when an image could not be rendered and may not be a good alternative depending of the context.

For example if the clock image doesn't load, the `clock` word appear. The user then will see `clock <remaining time>`. The image was nice as decoration, but now that it's gone, having the word **clock** in the UI seems weird.